<?xml version="1.0" encoding="UTF-8"?>
<launch>
  
    <param name="intrinsic_path"    value="$(find automatic-camera-pointcloud-calibration)/data/parameters/intrinsic.txt" />  <!-- intrinsic file -->
    <param name="input_photo_path"  value="$(find automatic-camera-pointcloud-calibration)/data/photo/5.jpg" />  <!-- photo to find the corner -->
    <param name="ouput_path"        value="$(find automatic-camera-pointcloud-calibration)/data/corner_photo.txt" />  <!-- file to save the photo corner -->
    
    <node pkg="automatic-camera-pointcloud-calibration" name="cornerPhoto" type="cornerPhoto" output="screen"></node>

    <node pkg="automatic-camera-pointcloud-calibration" name="cornerpoint" type="cornerpoint" output="screen"></node>

</launch>

